src/common/components/user-header/index.tsx

```diff
@@ -64,6 +64,17 @@ const UserHeader = (props: any) => {
               </div>
             </Row>
             <Row className="user-profile-feature-row">
+              <Col md={12} className="user-header-info" >
+              <div>
+                    <h3 className="text-capitalize mh-0">
+                      {metaProfile.name ? metaProfile.name : name}
+                    </h3>
+                    <h6 className="text-capitalize">ID: {id}</h6>
+                    {metaProfile.about && (
+                      <h6 className="text-capitalize">About: {metaProfile.about}</h6>
```

Comment:
all text should be added into translation string

Reply:
Updated in the translation files